stable/fluent-bit/templates/daemonset.yaml

```diff
@@ -103,6 +103,11 @@ spec:
         - name: mnt
           mountPath: /mnt
           readOnly: true
+{{ end }}
+{{ if .Values.on_microk8s }}
+        - name: varsnapmicrok8scommonvarlibdockercontainers
+          mountPath: /var/snap/microk8s/common/var/lib/docker/containers
```

Comment:
These paths seem wrong to me. I think the cri log spec has them looking in /var/lib/containers ?

Reply:
For MicroK8s they are located there: https://github.com/ubuntu/microk8s/issues/310

Reply:
yes, but i don't see anything in this PR that makes fluent-bit look for them in that location...

Based on the last comment on that issue, I think you need to pass through all 3 locations so that fluent-bit can find them at the cri logging location of /var/log/containers

Reply:
That is not my understanding. When using the current Helm chart, I can make it work on MicroK8s by adding the following the the values.yaml:

# Include the below if working on MicroK8s !
extraVolumes:
  - name: varsnapmicrok8scommonvarlibdockercontainers
    hostPath:
      path: /var/snap/microk8s/common/var/lib/docker/containers
extraVolumeMounts:
  - name: varsnapmicrok8scommonvarlibdockercontainers
    mountPath: /var/snap/microk8s/common/var/lib/docker/containers
    readOnly: true

A solution could also be to make this clear in the documentation instead and do the same for Minikube and then remove the options for both MicroK8s and Minikube?

Reply:
where does /var/log/containers point to in this case?